php/Terminus/Models/Upstream.php

```diff
@@ -37,7 +45,7 @@ public function fetch(array $args = [])
     {
         $options = array_merge(['options' => ['method' => 'get',],], $this->args, $args);
         $results = $this->request->request($this->url, $options);
-        $this->attributes = $this->parseAttributes($results['data']->upstream);
+        $this->attributes = (object)array_merge((array)$this->attributes, (array)$this->parseAttributes($results['data']->upstream));
```

Comment:
- No, it need to be much, much longer! Get the entire function on this line. ...Yes, less than.
- I don't think we should use parseAttributes. The concept I have in mind for it is that all it does is transform the data for use before assignment, but doesn't affect the class in any way.